src/lib/config.ts

```diff
@@ -82,6 +82,24 @@ export const AUTH_RESINOS_REGISTRY_CODE = optionalVar(
 	'AUTH_RESINOS_REGISTRY_CODE',
 );
 export const COOKIE_SESSION_SECRET = requiredVar('COOKIE_SESSION_SECRET');
+
+/**
+ * null: include all device type and device contract slugs
+ * "x;y;z": include only the specified device type and contract slugs - note that you MUST list
+ *          all dependent slugs as well so for hw.device-type/asus-tinker-board-s you would need
+ *          `armv7hf;asus;tinkerboard;asus-tinker-board-s` to pick up arch.sw/armv7hf,
+ *          hw.device-manufacturer/asus, hw.device-family/tinkerboard, and hw.device-type/asus-tinker-board-s
+ * 		    For something like hw.device-type/iot-gate-imx8 you would need `aarch64;iot-gate-imx8`
+ * 			to pick up arch.sw/aarch64 and hw.device-type/iot-gate-imx8
+ *          (the order of the slugs in this variable does not matter)
+ */
+export const INCLUDED_DEVICETYPE_SLUGS = new Map(
+	(optionalVar('INCLUDED_DEVICETYPE_SLUGS') ?? '')
+		.split(';')
+		.filter((slug) => slug.trim().length > 0)
+		.map((slug) => [slug.trim(), true]),
```

Comment:
NIT: Along with Page's suggestion on using a `Set`, we can also remove the duplicate `.trim()` by re-ordering the `.map` & `.filter`
```suggestion
		.map((slug) => slug.trim())
		.filter((slug) => slug.length > 0),
```